### PR TITLE
Code complexity

### PR DESCRIPTION
**Pull request by sritej-reddy**

### CODE REVIEW

**Comment by sritej-reddy** [comment]
closes #19

File: CodeComplexity/src/TimeComplexity.cpp [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+#include<math.h>
+
+char* function0(int n) {
+	//Sample Not tested
+	int i, j;
+	for (i = 0; i < n; i++) {
+
+	}
+
+	return "O(n)";
```
Comment (krushiraj) [inline CodeComplexity/src/TimeComplexity.cpp:32]: Please write comments before each return statement about why did you feel that the function has a certain time complexity, that whichever value you are returning. Give reasoning.

File: CodeComplexity/src/TwoSum.cpp [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+#include <stdio.h>
+#include <stdlib.h>
+#include<algorithm>
+using namespace std;
+
+/*
+Take each element, Sum with all other elements.
+Repeat it with all elements.
+
+What is the complexity of this method ? __________
```
Comment (krushiraj) [inline CodeComplexity/src/TwoSum.cpp]: Can you please fill these out?

**Review by krushiraj — CHANGES_REQUESTED** [review]
Please come up with the changes that I have requested in the comments

**Comment by sritej-reddy** [comment]
anna i have a question in function11 worst case should be n*n*n right? because worst case can be an array with all same elements.

**Comment by sritej-reddy** [comment]
I got answer👍👍

**Review by krushiraj — APPROVED** [review]
Everything is okay. Merging this.

**Pull request merged**